lib/pry-doc/pry_ext/show_source_with_c_internals/c_file.rb

```diff
@@ -0,0 +1,33 @@
+class CFile
+  SourceLocation = Struct.new(:file, :line, :original_symbol)
+
+  attr_accessor :symbols, :file_name
+
+  def self.from_str(str)
+    new(str).tap(&:process_symbols)
+  end
+
+  def initialize(str)
+    @lines = str.lines
+    @file_name = @lines.shift.split(",").first
+  end
+
+  def process_symbols
+    @symbols = @lines.map do |v|
+      symbol, line_number = v.split("\x7f")
```

Comment:
"\x7f" can potentially be a constant, so the name is descriptive. I have no idea what this is.